src/Text/Pandoc/Writers/Textile.hs

```diff
@@ -282,8 +282,7 @@ definitionListItemToTextile :: PandocMonad m
 definitionListItemToTextile opts (label, items) = do
   labelText <- inlineListToTextile opts label
   contents <- mapM (blockListToTextile opts) items
-  return $ "<dt>" <> labelText <> "</dt>\n" <>
-          T.intercalate "\n" (map (\d -> "<dd>" <> d <> "</dd>") contents)
+  return $ "- " <> labelText <> " := " <>) contents) $$ cr
```

Comment:
This is going to give you bad syntax if contents contains multiple paragraphs or other block level elements, won't it?  (See my original query on the issue.)

We could, as before, fall back to using a raw HTML definition list in this case.

In any case, I'm surprised that no tests failed with this change.  Aren't we testing definition lists with the textile writer?

Reply:
I have no idea what I'm doing whatsoever, I just thought that this would be a nice jumping-off point.

Textile does implement a "close definition list" token, `=:`, but I think me implementing it wouldn't be 100% the best, I can try though.